### PR TITLE
Routes/post user register

### PR DESCRIPTION
**Pull request by apiwonska**
Adding route for users POST (user register)

### CODE REVIEW

**Comment by apiwonska** [comment]
Please, wait with review I'm making changes

**Comment by apiwonska** [comment]
Please review

File: routes/users.js [hunk 1]
```diff
@@ -1,16 +1,40 @@
...
+} = require("../models/user");
+const mongoose = require("mongoose");
+const express = require("express");
+const bcrypt = require("bcryptjs");
 const router = express.Router();
 
 router.get('/', (req, res) => {
     res.render('registration', {
         errors: ["Test error message 1", "Test error message 2"]
     });
```
Comment (ireshka) [inline routes/users.js]: Project doesn't build. There is propably a problem with brackets here.

**Comment by apiwonska** [comment]
> Project doesn't build - there is propably a problem with brackets.

Fixed with last commits

**Pull request merged**